repository/managers.py

```diff
@@ -25,6 +28,8 @@ def _get_headers(self):
         try:
             return auth.citesphere_auth(self.user, self.repository)
         except Exception as e:
+            error_trace = traceback.format_exc()
+            logger.error(f"Error:\n{error_trace}")
```

Comment:
I think I've made this comment before somewhere, either handle or throw an exception. Since it is thrown here, nothing else needs to be done. It should be logged where it is being handled. (same for all the other cases).